js/bitfinex.js

```diff
@@ -658,7 +658,7 @@ module.exports = class bitfinex extends Exchange {
         return this.parseOrder (result);
     }
 
-    async replaceOrder (id, symbol, type, side, amount, price = undefined, params = {}) {
+    async editOrder (id, symbol, type, side, amount, price = undefined, params = {}) {
```

Comment:
There's still one more issue with the amount – it isn't required in most cases of editOrder. Therefore we have to properly account for that when we prepare the request. In other words, I wouldn't unify `prepareRequest` for `createOrder` and `editOrder` and would rather keep them separate, or it has to be edited further to account for an undefined amount.